apps/payment/models.py

```diff
@@ -33,26 +33,46 @@ class Payment(models.Model):
     )
 
     content_type = models.ForeignKey(ContentType)
+    """Which model the payment is created for. For attendance events this should be event."""
     object_id = models.PositiveIntegerField()
+    """
+    Object id for the model chosen in content_type.
+    For events this should be the event id and not attendance event id.
+    """
     content_object = GenericForeignKey()
+    """Helper attribute which points to the object select in object_id"""
     stripe_key = models.CharField(
         _('stripe key'),
         max_length=10,
         choices=STRIPE_KEY_CHOICES,
         default="arrkom"
     )
+    """Which Stripe key to use for payments"""
 
     payment_type = models.SmallIntegerField(_('type'), choices=TYPE_CHOICES)
+    """
+        Which payment type to use for payments.
+        Can be one of the following:
+
+        - Immidiate
+        - Deadline. See :attr:`deadline`
+        - Delay. See :attr:`delay`
+    """
 
-    # Optional fields depending on payment type
     deadline = models.DateTimeField(_("frist"), blank=True, null=True)
+    """Shared deadline for all payments"""
     active = models.BooleanField(default=True)
+    """Is payment activated"""
     delay = models.SmallIntegerField(_('utsettelse'), blank=True, null=True, default=2)
+    """Number of days before user has to pay"""
```

Comment:
Could probably add "after attending", as in "Number of days after user attended which they have to pay."